src/parser/shared/normalizers/FyralathNormalizer.ts

```diff
@@ -0,0 +1,93 @@
+import SPELLS from 'common/SPELLS';
+import { AnyEvent, BeginChannelEvent, EndChannelEvent, EventType } from 'parser/core/Events';
+import EventsNormalizer from 'parser/core/EventsNormalizer';
+import InsertableEventsWrapper from 'parser/core/InsertableEventsWrapper';
+
+/**
+ * Normalizes Rage of Fyr'alath casts to include BeginChannel and EndChannel events.
+ */
+export class FyralathNormalizer extends EventsNormalizer {
```

Comment:
is there a reason that this is not simply a `buffChannelSpec` in the core `Channeling` normalizer?

Reply:
I tried around a little with it, but it didn't seem to work. The buff is a completely separate id than the cast? There's also cases where the buff would never be applied as well

Reply:
hmmm okay